docs/concepts/what-is-a-codec-server.md

```diff
@@ -0,0 +1,32 @@
+---
+id: what-is-a-codec-server
+title: What is a Codec Server?
+sidebar_label: Codec Server
+description: Explanation and implementation of a remote encryption/decryption server.
+---
+
+A Codec Server is an optional implementation for your [Temporal Platform](/docs/concepts/what-is-the-temporal-platform). With it, deserialized Payloads can have their data displayed on your WebUI.
```

Comment:
I'm inclined to change "Payloads" to "Payload objects", at least at this first instance, unless we'll have "Payloads" as a first-class Temporal term. (Or lowercase "Payload[s]".) However, I'm not the expert. @flossypurse, any opinion?

Reply:
On the other hand.
Might be a worthwhile term for us.